.github/workflows/ci-quick-check.yml

```diff
@@ -8,8 +8,7 @@ on:
       - '*'
   pull_request:
     branches:
-      - master
-      - main
+      - '*'
```

Comment:
Here is the same change as before, the intention is to be able to have branches that work on bigger changes and can receive PRs that are checked by the CI. (Not currently used)